### PR TITLE
GCS and BQ locators and actions change

### PR DESCRIPTION
**Pull request by sawantpritam**
@itsankit-google @rmstar Please review.

1. Cleaned up xpaths of existing locators to remove class.
2. Added locators and actions.

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java [hunk 1]
```diff
@@ -133,4 +129,47 @@ public static void enterTemporaryBucketName(String bucket) throws IOException {
   public static void clickPreviewData() {
     SeleniumHelper.waitAndClick(CdfBigQueryPropertiesLocators.previewData);
   }
+
+  public static void enterPartitionField(String partitionField) {
+    CdfBigQueryPropertiesLocators.partitionField.sendKeys(partitionField);
```
Comment (itsankit-google) [inline src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java]: can we use `ElementHelper` in all the actions?

Comment (itsankit-google) [reply src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java]: This applies to all GCS actions as well.

Comment (sawantpritam) [reply src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java]: Element helper has additional checks and steps which are not required in all actions.
Element helper should be called only when necessary.

Comment (itsankit-google) [reply src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java]: Why we don't want these additional checks here? From my understanding, it checks for presence of element and scrolls to the element correctly which makes the test more stable, isn't it??

Comment (itsankit-google) [reply src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java]: @poojantcs WDYT?

Comment (poojantcs) [reply src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java]: Hi @itsankit-google , we should use the ElementHelper as much as we can. For instances where the ElementHelper is not working, we should look to either enhance the existing functions or add new ones to it.

Comment (sawantpritam) [reply src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java]: Updated to use ElementHelper.

File: src/main/java/io/cdap/e2e/pages/locators/CdfGCSLocators.java [hunk 2]
```diff
@@ -124,4 +107,19 @@
...
+
+  public static WebElement pathFilenameOnly(String value) {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//input[@name='filenameOnly' and @value='" + value + "']"));
+  }
+
+  public static WebElement recursive(String value) {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//input[@name='recursive' and @value='" + value + "']"));
+  }
+
+  public static WebElement getListItem(String option) {
```
Comment (itsankit-google) [inline src/main/java/io/cdap/e2e/pages/locators/CdfGCSLocators.java]: Is it a general locator or selecting item from particular list?? From locator name it is not clear. If its generic it should be in some general PluginPropertiesLocators.

Comment (sawantpritam) [reply src/main/java/io/cdap/e2e/pages/locators/CdfGCSLocators.java]: This is currently used only in GCSActions but can be used across plugins to get dropdown list item.
I will rename it and add TODO to move it to CdfPluginPropertiesLocators.java as this file is not yet merged.

**Review by itsankit-google — APPROVED** [review]
LGTM, Please squash commits.

File: src/main/java/io/cdap/e2e/pages/locators/CdfGCSLocators.java (hunk 2 above)
Comment (sawantpritam) [reply src/main/java/io/cdap/e2e/pages/locators/CdfGCSLocators.java]: Done.

**Comment by sawantpritam** [comment]
Squashed commits.

**Pull request merged**